Infrastructure/Extensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Infrastructure
+{
+    public static class Extensions
+    {
+        public static IServiceCollection AddContext(this IServiceCollection services, string connectionString)
```

Comment:
AddContext is not a screaming name, you are doing MySQL staff inside, which is implicit and not obvious enough